libs/remix-core-plugin/src/lib/compiler-content-imports.ts

```diff
@@ -22,7 +22,12 @@ export class CompilerImports extends Plugin {
 
   async setToken () {
     const protocol = typeof window !== 'undefined' && window.location.protocol
-    const token = await this.call('settings', 'get', 'settings/gist-access-token')
+    let token
+    try {
+      token = await this.call('settings', 'get', 'settings/gist-access-token')
```

Comment:
if this fails, you might not want to do `this.urlResolver.setGistToken(token, protocol)`